demo/mixin-component.html

```diff
@@ -0,0 +1,50 @@
+<link rel="import" href="../../polymer/polymer-element.html">
+<link rel="import" href="../d2l-localize-mixin.html">
+
+<dom-module id="d2l-mixin-component">
+	<template strip-whitespace>
+		<style>
+			:host {
+				display: block;
+			}
+		</style>
+		[[localize('hello')]]
+	</template>
+
+	<script>
+		/**
+		* @extends {Polymer.Element}
+		* @appliesMixin D2L.PolymerMixins.LocalizeMixin
```

Comment:
Again, this `@appliesMixin` is needed so linter can trace things.  I think this will come up over and over.